src/js/jsx/shared/Label.jsx

```diff
@@ -25,10 +25,48 @@ define(function (require, exports, module) {
     "use strict";
 
     var React = require("react"),
-        classnames = require("classnames");
+        classnames = require("classnames"),
+        Fluxxor = require("fluxxor"),
+        FluxMixin = Fluxxor.FluxMixin(React),
+        ScrubbyMixin = require("js/jsx/mixin/Scrubby");
     
     var Label = React.createClass({
-        mixins: [React.addons.PureRenderMixin],
+        mixins: [FluxMixin, ScrubbyMixin],
+        
+        propTypes: {
+            size: React.PropTypes.string,
+            onScrub: React.PropTypes.func,
+            onScrubStart: React.PropTypes.func,
+            scrubby: React.PropTypes.bool
```

Comment:
We can probably get away with removing scrubby and just checking for existence of onScrub function (onScrubStart/onScrubEnd can be optional)